### PR TITLE
Add Triss Merigold - Custom Voiced Follower

### PR DESCRIPTION
**Pull request by IllusiveMan196**
Add mod, location, cleaning

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -23821,3 +23821,11 @@ plugins:
     clean:
       - crc: 0x2358D3DD
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
+
+  - name: 'TrissMerigoldW3.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/54724?/' ]
```
Comment (sibir-ine) [inline masterlist.yaml]: That `?` doesn't need to be there.

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/54724/' ]
```

File: masterlist.yaml [hunk 2]
```diff
@@ -23821,3 +23821,11 @@ plugins:
...
+    dirty:
+      - <<: *quickClean
+        crc: 0x8058BF5E
+        util: '[SSEEdit v4.0.4]'
+        itm: 2
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
    dirty:
      - <<: *quickClean
        crc: 0x8058BF5E
        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 2
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
This can go to `Encounters - Allies`.

**Comment by IllusiveMan196** [comment]
Moved this as well

**Review by sibir-ine — APPROVED** [review]

**Comment by sibir-ine** [comment]
Thanks!

**Pull request merged**